ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/PaymentDetailsForm.tsx

```diff
@@ -0,0 +1,83 @@
+import {useNavigation} from '@react-navigation/native';
+import React from 'react';
+import {useForm, SubmitHandler} from 'react-hook-form';
+import {Text, View, Pressable} from 'react-native';
+
+import CustomAmountInput from './components/customAmountInput/CustomAmountInput';
+import CustomButton from './components/customButton/CustomButton';
+import CustomInput from './components/customInput/CustomInput';
+import CustomMessageInput from './components/customMessageInput/CustomMessageInput';
+import {styles} from './payment-details-form.styles';
+import {PaymentDetailsFormData} from './payment-details-form.types';
+import {NavigationType} from '../../../navigation/navigation.types';
+
+const PaymentDetailsForm: React.FC = () => {
+  const navigation = useNavigation<NavigationType>();
+
+  const {
+    control,
+    handleSubmit,
+    formState: {errors},
+  } = useForm<PaymentDetailsFormData>({
+    defaultValues: {
+      ToPayee: 'John McCabe',
+      statementMessage: '',
+      payeeMessage: '',
+    },
+  });
+
+  const onSubmit: SubmitHandler<PaymentDetailsFormData> = data =>
+    console.log(data);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.inputBox1}>
+        <CustomInput
+          name="ToPayee"
+          title="To"
+          placeholder="Jack"
+          control={control}
```

Comment:
Could you see a way not adding `control` value for every input explicitly?
Maybe there is a way to use some sort of `FormProvider`, consuming it inside `Input` component through hook?